src/Host/Client/Impl/Host/RHost.cs

```diff
@@ -401,6 +401,15 @@ public sealed partial class RHost : IDisposable, IRExpressionEvaluator {
                                 _callbacks.DirectoryChanged();
                                 break;
 
+                            case "library":
+                                _callbacks.ViewLibrary();
```

Comment:
Or, since we need to explicitly call `OverrideFunction`, create a list of callbacks in RSession, and add the callback to the OverrideFunction as a parameter